lib/src/screens/exchange/exchange_page.dart

```diff
@@ -99,6 +99,14 @@ class ExchangePage extends BasePage {
   @override
   AppBarStyle get appBarStyle => AppBarStyle.transparent;
 
+  @override
+  Function(BuildContext)? get pushToNextWidget => (context) {
+    FocusScopeNode currentFocus = FocusScope.of(context);
+    if (!currentFocus.hasPrimaryFocus) {
+      currentFocus.focusedChild?.unfocus();
+    }
```

Comment:
wouldn't this line be enough?
```Dart
FocusScope.of(context).unfocus();
```

Reply:
No, this doesn't work in our case, most likely because we are using custom focusNodes.